src/main/java/seedu/address/model/person/PersonSearchPredicate.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Tests that a {@code Person}'s {@code Name}, {@code Tags} or {@code Assets} matches any of the keywords given.
+ */
+public class PersonSearchPredicate implements Predicate<Person> {
```

Comment:
May I suggest using the name `PersonMatchesKeywordPredicate` as it describes what the class is meant to do.

Reply:
Renamed the class to `PersonMatchesKeywordPredicate`.